Wordle.Api/Wordle.Api/Dtos/PlayerDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Wordle.Api.Dtos;
+
+public record PlayerDto

```

Comment:
I had never really even heard of DTOs before I saw the way you did this - nice work!

Reply:
Thank you! :)